<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomItem</class>
 <widget class="QDialog" name="RoomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 213, 213);</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>五子棋-01房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_join">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 213, 213);</string>
   </property>
   <property name="text">
    <string>加入</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_player1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/icon/slotwait.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_player2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/icon/slotwait.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
